startup/31-xspress3.py

```diff
@@ -354,7 +354,11 @@ def mode(self):
 
     def describe(self):
         res = super().describe()
-        res['xs_fluor']["chunks"] = (1, "auto", -1, -1)
+        if self.mode == SRXMode.fly:
+            res['xs_fluor']["chunks"] = (1, "auto", -1, -1)
+        elif self.mode == SRXMode.step:
+            res['xs_fluor']["chunks"] = ("auto", -1, -1)
```

Comment:
@gwbischof I can't figure out how to modify your PR. Can you please remove these two lines of code and then it will match my local copy of your PR, which is tested and working at the beamline? Then we can merge this and the merge will properly reflect the changes at the beamline.